security/Security.php

```diff
@@ -147,6 +147,14 @@ class Security extends Controller implements TemplateGlobalProvider {
 	private static $frame_options = 'SAMEORIGIN';
 
 	/**
+	 * Value of the X-Robots-Tag header (for the Security section)
+	 *
+	 * @config
+	 * @var string
+	 */
+	private static $robots_tag = 'noindex, nofollow';
```

Comment:
I'd read somewhere that the more explicit version has better browser support, will see if I can find it